ui/src/components/ui/alert-dialog.tsx

```diff
@@ -7,32 +7,32 @@
  * SPDX-License-Identifier: MIT
  */
 
-import * as React from "react"
-import * as AlertDialogPrimitive from "@radix-ui/react-alert-dialog"
+import * as React from 'react';
```

Comment:
Just an observation: Interesting that apparently more files than just those in `api/queries` were not formatted correctly by the `codegen`'s `--format` option.

Reply:
My guess is that `codegen`'s `--format` only applied to the generated files (though it missed some), and not other files in the project.